client/my-sites/checkout/checkout-thank-you/test/index.js

```diff
@@ -105,129 +98,6 @@ describe( 'CheckoutThankYou', () => {
 		} );
 	} );
 
-	describe( 'Simplified page', () => {
```

Comment:
The redesigned congrats page doesn't have a "simplified" view. This test is no longer relevant.